### ISSUE TITLE
Eli5 doesn't play nicely with jupyter themes dark mode

### ISSUE THREAD

**Issue by BrunoGomesCoelho** [TeamHG-Memex/eli5#312]
I use [jupyter themes](https://github.com/dunovank/jupyter-themes) for my daily coding, but unfortunately eli5 results can not be seen when using their dark mode (more specifically, monokai dark mode). This is the output of a simple script:

![image](https://user-images.githubusercontent.com/17727737/58033254-8c8d6480-7afa-11e9-9e5a-8e853586c0d5.png)

Does anyone know how I could go around to changing the default visualization so it won't break with jupyter themes? I have posted the same question on their repo as well for ideas.

**Comment by lopuhin** [comment]
Oh that does not look pretty indeed :) I guess it's our job to expose some color scheme settings on eli5 side.

**Comment by shivendra90** [comment]
Saw the same while doing some diagnosis in JL. Some dark mode adjustments would be great.

**Comment by miguelamavel** [comment]
This is by no means a solution, but I've worked around this problem by changing the html directly:

![image](https://user-images.githubusercontent.com/20721873/74442532-f2cccb00-4e71-11ea-890f-206c0a6e0674.png)
